cli/command/service/opts.go

```diff
@@ -457,6 +457,33 @@ func convertExtraHostsToSwarmHosts(extraHosts []string) []string {
 	return hosts
 }
 
+type isolationOpts struct {
+	value  *container.Isolation
+	source string
+}
+
+func (o *isolationOpts) Set(value string) error {
+	o.source = value
+	tv := container.Isolation(strings.ToLower(value))
```

Comment:
I think we should remove the `strings.ToLower()`, and only accept strict values; other options are case-sensitive as well;

```bash
$ docker service create --mode=gLoBaL nginx:alpine
Unknown mode: gLoBaL, only replicated and global supported
```